File: js/fibonacci.js
const fibonacci = (num) => {
    let a = 0, b = 1, sum = 0;

     for(let i = 1; i < num; i++){ //Loop to establish fibonacci sequence using arbitrary number '100'
        sum = a + b;
        a = b;  //set variables for next loop of fibonacci sequence
        b = sum;

    }
    return sum;
}
module.exports = {fibonacci}

File: python/fibonacci.py
def fibonacci(n):
  sum = 0
  first_Num = 0
  second_Num = 1
 
  for i in range(1,n):
      sum = first_Num + second_Num
      first_Num = second_Num
      second_Num = sum
      
      # print(sum)
  return sum
print(fibonacci(45))